### PR TITLE
Setup mobile portfolio

### PR DESCRIPTION
**Pull request by tonnytech**
**Added the following changes to this branch:**

1. A .gitignore file
2. A linter configuration
3. A index.html file and used it to design the project
4. Styles.css file and used it to style the index.html file
5. Test.md file
6. A .hintrc file
7. stylintrc.jason file
8. Package.jason configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## How to use it
+
+Use your command line to clone this repository to your local computer.
+
+## Author
+ Name: Tonny Tei
+ Twiter: https://twitter.com/home?lang=en
+
+ ### Contributing
+Contributions, issues, and features requests are welcome.
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure your project is documented in a professional way.
- [ ] The description of the project is supposed to go in the description [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Estimated%20time%3A%204.5h-,Description,-For%20the%20third)section of your README
- [ ] Kindly make sure you use your social media handler correctly. 
e.g [tonny](https://the_actually_link_to_your_social_media _profil.com)

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  font-family: 'poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding-left: 24px;
+  padding-right: 24px;
```
Comment (christianbiring1) [inline styles.css]: - [ ] Kindly make sure you use the correct  `background-color` as the template provided in Figma

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# My portfolio project
+This is a porfolio project that has my details and infromation.
```
Comment (christianbiring1) [inline README.md]: - [ ] Kindly make sure the description of your project goes here. You can see the [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Estimated%20time%3A%202.5h-,Description,-For%20the%20first)

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @teitonny  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ]  All linters tests are green
- [ ] Descriptive commit message

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Put the description of the changes you make in your PR in the description bpx instead of listing the files you have in the repos.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — CHANGES_REQUESTED** [review]
Hi @teitonny , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a project with linter issues, Kindly fix them 🤗

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @teitonny , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the required changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**